### ISSUE TITLE
Add term oligosyndactyly

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10334]
**Preferred term label:**
Oligosyndactyly

**Synonyms**

**Definition (free text, please give PubMed ID)**
Combined syndactyly and oligodactyly of the hand or feet

**Parent term (use hpo.jax.org/app)**
Abnormal digit morphology HP:0011297

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM246570: FIBULAR APLASIA, TIBIAL CAMPOMELIA, AND OLIGOSYNDACTYLY SYNDROME

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
This is a bunded term. Please instead use the corresponding terms for oligodactyly and syndactyly!
